src/SemiclassicalOrthogonalPolynomials.jl

```diff
@@ -48,9 +48,9 @@ function getindex(P::SemiclassicalJacobiWeight, x::Real)
 end
 
 function sum(P::SemiclassicalJacobiWeight{T}) where T
-    (t,a,b,c) = map(big, map(float, (P.t,P.a,P.b,P.c)))
-    # (t,a,b,c) = P.t, P.a, P.b, P.c
-    return convert(T, t^c * beta(1+a,1+b) * _₂F₁general2(1+a,-c,2+a+b,1/t))
+    (t,a,b,c) = (P.t, P.a, P.b, P.c)
+    t,a,b,c = BigFloat("$t"),BigFloat("$a"),BigFloat("$b"),BigFloat("$c") # This is needed at high parameter values
```

Comment:
why would you do `BigFloat("$t")`? that is crazy. Use `convert(BigFloat, t)`

Reply:
We can do what you suggest but it loses some accuracy without really affecting speed too much.

Reply:
Note

```julia
julia> t = 1.1
1.1

julia> BigFloat("1.1")-convert(BigFloat,t)
-8.881784197001252323389053344726562499999999999999999999999999654553257796222215e-17

julia> BigFloat("1.1")-BigFloat("$t")
0.0
```

and this really matters here.

Reply:
This is nonsense. Here t is not 1.1 but fl(1.1) ie the rounded version. The fact it happens to print out as 1.1 is a coincidence.

Reply:
So what you are doing is losing accuracy because it’s rounding a base-2 number to base-10

Reply:
Well, it is only coincidence if we ignore that t is a user input number but I don't mind changing it, the truth is that if a user, i.e. we, really properly needs accuracy on these they should construct the polynomials with BigFloat in the first place.

Reply:
How on earth would we know the user had originally typed in 1.1?

And your approach is introducing error extra error if t = 1/3. That is you round to base 2 and then round again to base 10, then round to base 2 with higher precision.

Reply:
fixed